pkg/sql/tenant.go

```diff
@@ -462,15 +462,23 @@ func GCTenantSync(ctx context.Context, execCfg *ExecutorConfig, info *descpb.Ten
 		}
 
 		// Clear out all span config records left over by the tenant.
-		tenantPrefix := keys.MakeTenantPrefix(roachpb.MakeTenantID(info.ID))
+		tenID := roachpb.MakeTenantID(info.ID)
+		tenantPrefix := keys.MakeTenantPrefix(tenID)
 		tenantSpan := roachpb.Span{
 			Key:    tenantPrefix,
 			EndKey: tenantPrefix.PrefixEnd(),
 		}
 
+		systemTarget, err := spanconfig.MakeTenantKeyspaceTarget(tenID, tenID)
```

Comment:
maybe a one-liner explaining what we're doing here?

Reply:
Done.